proto/options.proto

```diff
@@ -482,4 +482,5 @@ message Options {
     uint32 matrix_locations = 54;                                  // a one to many or many to one time distance matrix. Does not affect
   }                                                                // sources_to_targets when either sources or targets has more than 1 location
                                                                    // or when CostMatrix is the selected matrix mode.
+  bool blind_user_mode = 81;                           // route: additional maneuvers and informations for blind users.
```

Comment:
I was just looking at "vehicle types" and realized we already have smth you want in our request parameters, however, not documented. In PBF we have `transport_type` which is being set by either `bicycle_type` request param or `type` for all other modes, also pedestrian (not documented). Maybe we should use that instead of this boolean? Then we can make it a `TravelType` which will make more sense once we get to implement `tactile_paving` in costing. What do you think @grzezlo ?